contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -101,6 +107,15 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
     ///      protocol funds.
     uint256 public constant SOLVENCY_THRESHOLD = 0.998 ether;
 
+    /// @dev block interval within which the rebalancerVolumeLimit applies. 300 blocks equals ~1 hour
+    uint256 public constant REBALANCER_TRADE_INTERVAL_BLOCKS = 300;
+
+    /// @dev information regarding the trading volume of the rebalancer
+    struct RebalancerVolumeTally {
+        uint256 startBlock;
```

Comment:
Nit: Could make both uint128 and fit in one slot. Block number and swap volume are not really gonna overflow